development.ini

```diff
@@ -44,8 +44,9 @@ file_template = %%(year)d%%(month).2d%%(day).2d_%%(rev)s
 # file_template = %%(rev)s_%%(slug)s
 
 [server:main]
-use = egg:waitress#main
-listen = 0.0.0.0:8000
+use = egg:gunicorn#main
+host = 0.0.0.0
+port = 8000
```

Comment:
If we are going to keep waitress for compatibility, then we probably should also leave trace of how waitress can be configured here.  I would suggest we comment out old waitress config instead of deleting it here.